.wordlist.txt

```diff
@@ -0,0 +1,165 @@
+AAI
+accessionId
+addObjectToDrafts
+addObjectToFolder
+Ajv
+api
+apisauce
+asciicast
+asciinema
+Async
+async
+autenticate
+auth
+AUTHDB
+automodule
+autosummary
+backend
+baseURL
+CERTREQS
+CLI
+conf
+const
+createFromJSON
+createFromXML
+createNewDraftFolder
+createSlice
+CSC
+CSCfi
+cscfi
+css
+currentSubmissionType
+DAC
+Dataset
+destructure
+ebi
+EGA
+ega
+ENA
+ena
+enasequence
+ENV
+env
+eppn
+eslint
+eslintrc
+FhJ
+fileName
+folderID
+followingly
+FormData
+formData
+frontend
+FWYs
+genindex
+Github
+github
+githubusercontent
+GroupedBySchema
+gunicorn
+hostname
+href
+html
+http
+https
+INITDB
+inteded
+io
+javascript
+js
+JSON
+json
+JSONContent
+JWK
+Keyfile
+Kubernetes
+li
+localhost
+lt
+makeStyles
+maxdepth
+md
+metadataObjects
+Metdata
+middlewares
+modindex
+mongo
+Mongodb
+mongodb
+mTFEFsWsNUbP
+Nav
+NeIC
+neic
+newdraft
+noindex
+npm
+npx
+objectDetails
+ObjectInsideFolder
+ObjectInsideFolderWithTags
+ObjectStatus
+ObjectSubmissionTypes
+ObjectTags
+objectType
+ObjectTypes
+OIDC
+ol
+OpenID
+openid
+overriden
+phenome
+pre
+prettierrc
+ProviderMetadata
+py
+PyCQA
+pycqa
+quickstart
+readthedocs
+redux
+REFERER
+reqs
+rootReducer
+rst
+schemas
+SDA
+sda
+seprately
+SLLVERSION
+spesified
```

Comment:
```suggestion
specified
```

Reply:
u don't see this anywhere in the code, is this needed ?

Reply:
took away irrelevant string from wordlist